camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java

```diff
@@ -560,8 +561,14 @@ public int run(String[] args) throws Exception {
 
     props.putAll(cmd.getOptionProperties("D"));
 
-    run();
-    return 0;
+    try {
+      run();
+      log.info("Exiting successful Camus run.");
+      return 0;
+    } catch(Exception ex) {
```

Comment:
I forgot what was the reason that you have to catch this exception here (instead of letting this exception throw popped up through the stack?)